README.md

```diff
@@ -38,7 +38,7 @@ date, then clone the dataprofiler from the feature branch you want in order to
 update the documentation (probably main).
 
     # clone the feature branch into the base of this repository
-    git clone https://github.com/capitalone/DataProfiler DataProfiler
+    git clone https://github.com/capitalone/DataProfiler feature_branch
```

Comment:
switched to this mechanism otherwise `DataProfiler` could cause trouble when switching between branches.